test/tasty/Data/Aeson/CommitTest.hs

```diff
@@ -27,19 +31,20 @@ tests = testParserWithCases pNested
     )
   , ( "fails on malformed nested"
     , [aesonQQ| { value: "top", nested: { foo: 9 } } |]
-    , Left "Error in $.nested: key \"value\" not present"
+    , Left "Error in $.nested: key \"value\" not found"
```

Comment:
Maybe it's due to different versions of `aeson` we pull in because the `.cabal` file is too lax. Maybe we should specify a more specific version range?

Reply:
I used the shell from `bryggja`. And with the version of `aeson` pulled in from there I do indeed get `present` so I reverted the test to assert that instead.

Reply:
Yeah, I'm in favour of reigning in the versions for this. Older versions don't even have a lot of the `<?>` stuff we use here and completely fail to compile. When I was testing things a few weeks ago I had to ensure an Aeson version of at least 1.4.5.0.

Reply:
I just verified that `present` is used in aeson-1.4.5.0 but from aeson-1.4.6.0 the word `found` is used instead.